config/routes.rb

```diff
@@ -4,4 +4,5 @@
   end
 
   resources :scores, only: [:index, :new, :create, :destroy]
+  resources :requests, only: [:create, :destroy]
 end
```

Comment:
I considered going the route of updating requests rather than destroying. This feels a bit cleaner & will certainly keep that table smaller. I see no reason for a request to collaborate to remain in the database after it's been dealt with - but open to adjusting this if neccessary.

Reply:
yeah that makes sense to let go of the records after we don't need them anymore